common/djangoapps/edxmako/shortcuts.py

```diff
@@ -151,6 +152,13 @@ def render_to_string(template_name, dictionary, namespace='main', request=None):
         request: The request to use to construct the RequestContext for rendering
             this template. If not supplied, the current request will be used.
     """
+    responses = signals.BEFORE_RENDER_TO_RESPONSE.send(sender=template_name, template_name=template_name, dictionary=dictionary, request=request)
+    for receiver, response in responses:
+        if isinstance(list, response):
```

Comment:
I'm not following what is going on here. Why are you checking if `list` is an instance of `response` and if so treating response like it is a dictionary?

Reply:
Sorry, fixed.